cli/Flags.cpp

```diff
@@ -87,4 +87,15 @@ DEFINE_bool(preload_buffer_pool, false,
             "accepting queries (should also set --buffer_pool_slots to be "
             "large enough to accomodate the entire database).");
 
+static bool ValidatePort(const char *flagname, std::int32_t value) {
+  if (value > 0 && value < 65536) {
+    return true;
+  }
+  std::cout << "Invalid value for --" << flagname << ": " << std::to_string(value) << std::endl;
```

Comment:
Should the error message also include the valid range of values? That might be helpful.